### DISCUSSION TITLE
Consistencia en Traducciones

### DISCUSSION THREAD

**Discussion by J-Ciro** [reactjs/es.reactjs.org#672] [Q&A] [answered]
Me gustaría que se ampliara el glosario ya que muchas veces se vuelve inconsistente las traducciones, por ejemplo, **See more examples below** hay muchas variaciones, y siento que sería mejor que se ampliara el glosario para que todo esté más "standar" y palabras como estas sean consistentes en las traducciones

![image](https://user-images.githubusercontent.com/37349818/235515490-10ffe7b2-cf25-4ed0-b6f3-68faa950bcef.png)
![image](https://user-images.githubusercontent.com/37349818/235515522-4e877f96-c156-4567-955e-6ea37f2c0ae4.png)
![image](https://user-images.githubusercontent.com/37349818/235515547-f8101dd3-d226-4464-8169-1f5eef1d8c43.png)
![image](https://user-images.githubusercontent.com/37349818/235515666-dfc1d3d8-ff2f-4466-beb3-5cddcfb20227.png)

**Comment by mateoguzmana** [comment]
Buen punto @J-Ciro 

En mi opinión, «Ver más ejemplos» podría ser el estándar más adecuado. Las otras dos alternativas no son tan comunes en la documentación en castellano en general.

**Comment by luiserdef** [comment]
Este es un detalle que también me di cuenta cuando empezaba a traducir páginas y antes de hacer algo miraba como estaban traducidas otras páginas, esto para mantener la consistencia. Al final me tocó elegir las que me parecía la mejor. Algo igual pasa por ejemplo con la palabra "returns". 
Por lo que lo que he leído cuando se termine de traducir las páginas se corregirá.

**Comment by NGHdeveloper** [comment, marked as answer]
Ya fue observado y se hicieron los cambios pertinentes en el pull request [Consistencia en Traducciones #801](https://github.com/reactjs/es.react.dev/pull/801), segun indica el [Glosario](https://github.com/reactjs/es.react.dev/wiki/Glosario#secciones-comunes)
